pkg/webhook/v1beta1/pod/inject_webhook.go

```diff
@@ -259,7 +267,7 @@ func (s *SidecarInjector) getKatibJob(object *unstructured.Unstructured, namespa
 			// Get group and version from owner API version
 			gv, err := schema.ParseGroupVersion(owners[i].APIVersion)
 			if err != nil {
-				return "", "", err
+				return "", "", fmt.Errorf("%w: %s", errInvalidOwnerAPIVersion, err.Error())
```

Comment:
```suggestion
				return "", "", fmt.Errorf("%w: %w", errInvalidOwnerAPIVersion, err)
```

Reply:
This has been fixed in this [commit](https://github.com/kubeflow/katib/pull/2289/commits/e0518aca1cb4b33693b617285abd049f1d5e0e23).